ingestion/functions/parsing/riograndedosul/riograndedosul_test.py

```diff
@@ -0,0 +1,50 @@
+import csv
+import json
+import os
+import tempfile
+import unittest
+from datetime import date
+
+import pytest
+
+from riograndedosul import riograndedosul
+
+_SOURCE_ID = "abc123"
+_SOURCE_URL = "https://ti.saude.rs.gov.br/covid19/download"
+_PARSED_CASE = (
+    {'caseReference': {'sourceId': 'abc123', 'sourceUrl': 'https://ti.saude.rs.gov.br/covid19/download'},
+     'location': {'query': 'PORTO ALEGRE, Rio Grande do Sul, Brazil'},
+     'events': [{'name': 'confirmed',
+                 'dateRange': {'start': '06/18/2020Z', 'end': '06/18/2020Z'}},
+                {'name': 'onsetSymptoms',
+                 'dateRange': {'start': '06/02/2020Z', 'end': '06/02/2020Z'}},
+                {'name': 'hospitalAdmission',
+                 'dateRange': {'start': '06/18/2020Z', 'end': '06/18/2020Z'},
+                 'value': 'Yes'},
+                {'name': 'outcome',
+                 'dateRange': {'start': '07/08/2020Z', 'end': '07/08/2020Z'},
+                 'value': 'Death'}],
+        'demographics': {'gender': 'Male',
+                         'ageRange': {'start': 40.0, 'end': 49.0},
+                         'ethnicity': None},
```

Comment:
To me it looks as though, in sample_data, the ethnicity ('RACA_COR') is 'BRANCA' and that the information about indigenous ethnicity ('ETNIA_INDIGENA') is 'NAO ENCONTRADO'. I think you may need to check your indexing.

Reply:
Great catch! Thanks for this! In future parsers, I think I'm going to switch over to csv.DictReader to avoid indexing issues. It's a pretty easy error for me to make, and that just removes it from the equation.

Reply:
Agreed, just made that change for Paraiba, will switch over for new parsers too